recipes/pcapplusplus/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "PcapPlusPlus" %}
+{% set version = "22.05" %}
+{% set sha256 = "5f299c4503bf5d3c29f82b8d876a19be7dea29c2aadcb52f2f3b394846c21da9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/seladb/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+  host:
+  - libpcap
+  run:
+  - libpcap
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/pcapplusplus/PcapPlusPlusVersion.h
+    - test -f ${PREFIX}/lib/libCommon++.a
+    - test -f ${PREFIX}/lib/libPacket++.a
+    - test -f ${PREFIX}/lib/libPcap++.a
```

Comment:
Please build shared libraries, we don't support static linkage as the primary path.

Reply:
I agree with you, but I didn't find a way to build shared libs. it's seems to be a requested feature for a long time:
https://github.com/seladb/PcapPlusPlus/issues/61

Reply:
Yes, this doesn't look simple :(